src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+// ACTIONS
+const ADD = 'my-app/widgets/ADD';
+const REMOVE = 'my-app/widgets/REMOVE';
```

Comment:
- [ ] Kindly rename the actions to reflect the structure of the project. Renaming actions these ways reflect the duck pattern and make it easy to read your code. I recommend you follow the example of the first reviewer. 👍 You can read more on duck patterns by clicking this [link](https://github.com/erikras/ducks-modular-redux).